gcc/testsuite/rust/compile/break-rust2.rs

```diff
@@ -1,4 +1,4 @@
 fn main() {
     break (rust);
-    // { dg-error "failed to find name: rust" "" { target *-*-* } .-1 }
+    // { dg-error "cannot find value `rust` in this scope" "" { target *-*-* } .-1 }
```

Comment:
using "cannot find value .rust. in this scope" using '.' is a regex that just matches to a character